src/components/TopBar/TopBar.vue

```diff
@@ -125,21 +125,42 @@
 				close-after-click="true"
 				container="#content-vue">
 				<ActionButton
+					v-if="isInCall"
+					key="openSideBarButtonMessageText"
+					@click="openSidebar">
+					<MessageText
+						slot="icon"
+						:size="24"
```

Comment:
```suggestion
						:size=16"
```

like all other icons in the UI?